src/App.vue

```diff
@@ -115,6 +132,11 @@ export default {
       if (value) this.$Progress.start();
       else this.$Progress.finish();
     },
+    loginStatus(newValue, oldValue) {
+      // if user was logged in before but has been logged out now
+      // show a popup telling the user that they're logged out
+      if (oldValue && !newValue) this.showUserLoggedOutDialog();
```

Comment:
would have to be done using isAuthentication.
Also, this is under watch - so, the old Value and newValue will necessarily be different. 
Given it is a boolean - as long as newValue is False, that check is enough.